docs/pages/_app.js

```diff
@@ -1,19 +1,114 @@
-import React from 'react'
+import React from 'react';
 import * as Sentry from '@sentry/browser';
-import App, { Container } from 'next/app'
+import App, { Container } from 'next/app';
 
-Sentry.init({dsn: "https://67e35a01698649d5aa33aaab61777851@sentry.io/1526800"});
+Sentry.init({
+  dsn: 'https://67e35a01698649d5aa33aaab61777851@sentry.io/1526800',
+  beforeSend(event) {
+    let message = getMessage(event);
 
-class MyApp extends App {
+    // If we don't know about this particular type of event then just pass it along
+    if (!message) {
+      return event;
+    }
+
+    // Discard any errors that we know we do not care about
+    if (ERRORS_TO_DISCARD.includes(message)) {
+      return null;
+    }
+
+    // Only attempt to check against cached reported messages if we have localStorage
+    if (window.localStorage) {
```

Comment:
localStorage throws issues in private browsing mode, not sure it's worth dealing with - https://stackoverflow.com/questions/14555347/html5-localstorage-error-with-safari-quota-exceeded-err-dom-exception-22-an